### PR TITLE
272 SIM - BE - Fix - json load for validator config 

### PR DESCRIPTION
**Pull request by bradleySuira**
This pr includes a hot fix for validator config parsing

### CODE REVIEW

**Review by AgustinRamiroDiaz — CHANGES_REQUESTED** [review]
We've talked with Bradley outside, and I don't think this is the right way to tackle this issue. I'll put more information on the issue

**Comment by bradleySuira** [comment]
> We've talked with Bradley outside, and I don't think this is the right way to tackle this issue. I'll put more information on the issue

pushed the fixes and updated the ticket with the details but want to add here

- As you said be were storing a string in a jsonb field in database
- I ended sending when call to create and update methods a valid json from the frontend
- in the backend side in both, update and create, I ended using this ` "config": AsIs("'" + json.dumps(config) + "'"),`, with this change the field is stored as a json in the database and not as string
- Because the last point, we don't need to cal anymore the json.loads() method when we get the validators data, so `json.loads(validator_data["config"]),`  is now `validator_data["config"]` like we have before

I tested addling listing and updating the validators and also the contract deployment and methods calls

File: backend/database_handler/validators_registry.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def create_validator(
             "stake": stake,
             "provider": provider,
             "model": model,
-            "config": json.dumps(config),
+            "config": AsIs("'" + json.dumps(config) + "'"),
```
Comment (AgustinRamiroDiaz) [inline backend/database_handler/validators_registry.py:60]: @bradleySuira why can't we just directly use
```suggestion
            "config": config,
```
? From [the docs](https://www.psycopg.org/docs/extras.html#adapt-json) seems like it automatically adapts it out of the box. If not, it suggests to use the [Json adapter](https://www.psycopg.org/docs/extras.html#psycopg2.extras.Json)

**Review by AgustinRamiroDiaz — APPROVED** [review]

File: backend/database_handler/validators_registry.py (hunk 1 above)
Comment (bradleySuira) [reply backend/database_handler/validators_registry.py]: thanks @AgustinRamiroDiaz used at the end `from psycopg2.extras import Json` and works well

**Review by cristiam86 — APPROVED** [review]

**Pull request merged**